tests/util.py

```diff
@@ -0,0 +1,52 @@
+from uuid import uuid4
```

Comment:
These should be pytest fixtures.

Reply:
How would you solve this for instance? 
```
@pytest.mark.parametrize('transition_type,params', [
    ('docker', {'imageUrl': 'python3.8'}),
    ('manual', None),
    ('manual', {'assets': {'jsRemoteComponent': util.asset_id()}}),
])
def test_create_transition(client: BaseClient, transition_type, params, schema):
```

Reply:
Things connected to domain logic don't belong in utility modules. A simpler solution would be moving them to another module and prefixing them with `create_`.